### ISSUE TITLE
$variables are no more replaced where they're used

### ISSUE THREAD

**Issue by thepra** [failwyn/WebCompiler#11]
### Installed product versions
- Visual Studio: 2022
- This extension: 1.12.405

### Description
The dollar variables are no more replaced anywhere in a .scss file, breaking up everything in the pipeline.
(I'm using the NuGet package, at compilation time the WebCompiler is triggered.)

### Steps to recreate
1. Setup the code like this in `variables.scss`:
```
$main-pc: hsl(25,84,88); //#fbe9d7;

:root {
	--background: $main-pc;
}
```
2. Compile with this setup, where main.scss does `@import` of `variables.scss`:
```
[
	{
		"outputFile": "wwwroot/css/my-style.css",
		"inputFile": "SCSS/main.scss"
	}
]
```
3. Check the `my-style.css` output to find this:
```
:root {
	--background: $main-pc;
}
```

### Current behavior
It doesn't apply the dollar variables.

### Expected behavior
It should apply the dollars variable value.

**Comment by failwyn** [comment]
This looks like a duplicate of #4, can you confirm that this fixed your issue.

-- Comment copied from #4 --
It looks like this was caused by a change to node-sass

sass/node-sass#2627
sass/libsass#2621
https://sass-lang.com/documentation/style-rules/declarations#custom-properties

Can you try the following and let me know if it works?

$var-name: green;
p {
--my-custom-color: #{$var-name};
}
